docker-compose.yml

```diff
@@ -1,23 +1,26 @@
-version: '2'
+version: '2.1'
 
 services:
+  # Build: docker-compose build -d bdb
+  # Run: docker-compose run -d bdb
   mdb:
-    image: mongo:3.4.3
+    image: mongo:3.4.13
     ports:
       - "27017"
     command: mongod
-
-  bdb:
+  bigchaindb:
```

Comment:
nit-pick: `bdb` is easier to type when running the command and if you are changing this might want to consider changing `mdb` to `mongodb` as well for consistency

Reply:
already did.